### PR TITLE
README: give credit where credit is due

### PR DESCRIPTION
**Pull request by moneromooo-monero**

### CODE REVIEW

**Comment by hyc** [comment]
An older version of README.md 8e122401c9cc45b0ba511f4f085c96d4736a4ecb had this line

```+Parts of the project are originally copyright (c) 2012-2013 The Cryptonote developers```

but it was in the License section, which has subsequently been moved to its own LICENSE file. The above quoted line is still present in the LICENSE file so I believe proper credit is still being given.

**Comment by hyc** [comment]
I would amend this patch to say "Portions copyright 2012-2013 ..."

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Monero
 
 Copyright (c) 2014-2017, The Monero Project
+Copyright (c) 2012-2013, The Cryptonote developers
```
Comment (hyc) [inline README.md]: "Portions Copyright (c) 2012-2013, The Cryptonote developers"

**Comment by moneromooo-monero** [comment]
With portions now.

**Pull request merged**